graylog2-web-interface/src/components/permissions/RolesSelector.jsx

```diff
@@ -44,7 +44,7 @@ const _options = (roles, assignedRolesIds, identifier) => roles
   .toArray()
   .map((r) => ({ label: r.name, value: r.name, role: r }));
 
-const _assignRole = (selectedRoleName, roles, onSubmit, setSelectedRoleName, setIsSubmitting) => {
+const _assignRole = (selectedRoleName, roles, onSubmit, setSelectedRoleName, setIsSubmitting, setError) => {
```

Comment:
Lets reset the error somewhere, otherwise it will not get removed on a successful request.